ambry-store/src/main/java/com.github.ambry.store/StoreMetrics.java

```diff
@@ -133,6 +135,10 @@ public StoreMetrics(String prefix, MetricRegistry registry) {
     bloomPositiveCount = registry.counter(MetricRegistry.name(IndexSegment.class, name + "BloomPositiveCount"));
     bloomFalsePositiveCount =
         registry.counter(MetricRegistry.name(IndexSegment.class, name + "BloomFalsePositiveCount"));
+    indexSegmentInMemory =
+        registry.counter(MetricRegistry.name(IndexSegment.class, name + "IndexSegmentInMemoryCount"));
```

Comment:
`IndexSegmentInMemoryCount` is a bit misleading given what you are using this metric for. `IndexSegmentInMemoryDuringFind` or `...ForFind` is probably better names.

Reply:
Renamed to `IndexSegment(not)InMemoryDuringFindCount`.